library.properties

```
@@ -0,0 +1,11 @@
+name=Azure SDK for C
+version=1.0.0
+author=Microsoft Corporation
+maintainer=Microsoft <iotcert@microsoft.com>
+sentence=Azure SDK for C library (1.3.0-beta.2) for Arduino.
+paragraph=Arduino port of the Azure SDK for C. It allows you to use your Arduino with the Azure services like IoT Hub and Azure Device Provisioning Service.  See readme.md for more details.  Copyright (c) Microsoft. All rights reserved.  Licensed under the MIT license. See LICENSE file in the project root for full license information.
+category=Communication
```

👍  nice!

some nit (consistent spaces after period for example):
```suggestion
paragraph=Arduino port of the Azure SDK for C. It allows you to use your Arduino with Azure services like IoT Hub and Azure Device Provisioning Service. See README.md for more details. Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.
category=Communication
```

Also, should it be `Azure IoT Hub` or just `IoT Hub`?

---

All taken. Thx. And yes I used Azure IoT Hub.